### ISSUE TITLE
Error creating URL for IPv6 address

### ISSUE THREAD

**Issue by Revertron** [AdguardTeam/AdguardForAndroid#640]
```
ERROR com.adguard.filter.proxy.a - TCP id=152586 Error while processing connection. Connection info: com.instagram.android /fd12:1:1:1::1:46788 /2607:7700:0:1d::3456:3644:443 10129

java.net.MalformedURLException: java.lang.NumberFormatException: Invalid int: "7700:0:1d::3456:3644"
	at java.net.URL.<init>(URL.java:190) ~[na:0.0]
	at java.net.URL.<init>(URL.java:125) ~[na:0.0]
	at com.adguard.filter.proxy.a.a(Unknown Source) ~[na:0.0]
	at com.adguard.filter.proxy.a.c(Unknown Source) ~[na:0.0]
	at com.adguard.filter.proxy.a.a(Unknown Source) ~[na:0.0]
	at com.adguard.android.filtering.filter.f.a(Unknown Source) ~[na:0.0]
	at com.adguard.filter.c.d.a(Unknown Source) ~[na:0.0]
	at com.adguard.filter.c.d$2.run(Unknown Source) ~[na:0.0]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) ~[na:0.0]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) ~[na:0.0]
	at java.lang.Thread.run(Thread.java:818) ~[na:0.0]
```

**Comment by ameshkov** [comment]
We've already fixed this issue once, are you sure this is v2.6?

**Comment by Revertron** [comment]
Agrh, it's in 2.5.196
